feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt

```diff
@@ -141,7 +146,8 @@ fun SearchRoot(
                         selectedCategories = state.filter.selectedCategories,
                         categories = sheetState.categories,
                         onCategoriesSelected = viewModel::onCategoriesSelected,
-                        onDismiss = viewModel::onFilterSheetDismissed
+                        onDismiss = viewModel::onFilterSheetDismissed,
+                        contentPadding = WindowInsets.navigationBars.asPaddingValues(),
```

Comment:
I have a question 🙋.
Can we use content padding on all sheets? Do we need to use it differently when we have more or less content?

Reply:
This padding will be set to LazyList so if a sheet is composed only with LazyList, I think "yes".
But if we have to use other composables under the LazyList, we need to change here.

Reply:
That makes sense! Thanks!